user_util/user_util.py

```diff
@@ -1,3 +1,33 @@
 # -*- coding: utf-8 -*-
 
 """Main module."""
+import hashlib
+
+
+RETIRED_USERNAME_FMT = 'retired_username_{}'
+RETIRED_EMAIL_FMT = 'retired_email_{}@edx.org'
+USERNAME_HASH_LENGTH = 10
```

Comment:
Why 10? The Django char limit is 150, we should be able to fit the whole thing in there?

Reply:
I was wondering if the username would be showing up anywhere - and, if so, would such a long username foul up front-end things. Otherwise, fine with me for it to be long. I'll remove the limit.